packer/main.pkr.hcl

```diff
@@ -46,9 +46,12 @@ build {
 
   provisioner "shell" {
```

Comment:
Move download_node_exporter to packer while building image.
If done on TF -> Issue: no connection from github server.